core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-permissions/dot-container-permissions.component.scss

```diff
@@ -0,0 +1,4 @@
+dot-portlet-box {
+    height: 70vh;
```

Comment:
This is not responsive... we don't know if the rest of the elements of the page will be 30vh.

Reply:
@zulqarnainvd